package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Um bot de Discord que simula o Wordle",
   "main": "bot.js",
   "scripts": {
-    "deploy": "node src/update-commands.js",
     "dev": "tsnd --transpile-only src/bot.ts",
     "commands": "tsnd --transpile-only src/register.ts",
-    "start": "node src/bot.js",
+    "postinstall": "npm run compile",
+    "compile": "tsc",
```

Comment:
Heroku automatically runs the `build` script, so there's no need for `postinstall`.
```suggestion
    "build": "tsc",
```

Source: [Heroku Dev Center](https://devcenter.heroku.com/changelog-items/1557)

Reply:
Are you sure that is it ok to have only "tsc" in the script?

Reply:
I didn't test it. But it should work, at least in theory.